aion/lib/aion/room_channel/question_set.ex

```diff
@@ -31,19 +31,22 @@ defmodule Aion.RoomChannel.QuestionSet do
     end
   end
 
-  @spec new(Question.t, (list Question.t)) :: __MODULE__.t
+  @spec new(Question.t(), list(Question.t())) :: __MODULE__.t()
   defp new(current_question, remaining_questions) do
     answers = Answer.get_answers(current_question.id)
-    Logger.debug fn -> "Answers: #{inspect(Enum.map(answers, fn answer -> answer.content end))}" end
+
+    Logger.debug(fn ->
+      "Answers: #{inspect(Enum.map(answers, fn answer -> answer.content end))}"
+    end)
 
     %__MODULE__{
       questions: remaining_questions,
       current_question: current_question,
-      answers: answers,
+      answers: answers
```

Comment:
At first we were for, and now we are against trailing commas? (I guess its mix formatter but still)

Reply:
Yes, we were for because there was no official convention given from the lang. Right now there is mix formatter, which decides to remove trailing commas. It's possible that they will be accepted by the convention soon, but, at least for now, they are against it.